openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -185,13 +185,6 @@ def test_last_login_updated(self):
         self.user.refresh_from_db()
         assert self.user.last_login > old_last_login
 
-    def test_login_success_prevent_auth_user_writes(self):
```

Comment:
Can you remove the test above (`test_last_login_updated`) as well - since it's related and can be missed later when we go through to remove all code near `PREVENT_AUTH_USER_WRITES`?

Reply:
Ah! Yes, it's kind of pointless without the other one, isn't it.